packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -234,7 +235,7 @@ const SigninContainer = ({
       // warning. The browser will automatically respond with { ok: true } without
       // prompting the user if it matches the email the browser has data for.
       if (
-        integration.isSync() &&
+        isSyncDesktopV3Integration(integration) &&
```

Comment:
Also: I think we may want to check for the condition in that `firefox.ts` file so we don't have to worry about checking it before calling this. I just didn't do that here since we haven't passed `integration` into a `firefox.whatever` call yet and the approach might change given the linked follow up. (User agent for example seems fine to check in `firefox.ts`.)